apps/server/src/main.ts

```diff
@@ -25,6 +25,9 @@ async function bootstrap() {
   const document = SwaggerModule.createDocument(app, config);
   SwaggerModule.setup('api', app, document);
 
+  // @todo remove
+  app.enableCors()
```

Comment:
(should): use nginx as reverse proxy even if this is a quick solution

Reply:
this is for later if we develop with docker this should be no issue since nginx is allready running inside docker